File: poo/3. Newton's Laws Simulator/inertia.php
<?php 
    class inertia extends forces {
        private int $inertia;
        readonly private int $mass;

        public function __construct(int $mass) {
            parent::__construct("Inercia", "Inercia");
            $this->mass = $mass;
            $this->inertia = $mass;
        }
        public function getInertia(): int {
            return $this->inertia;
        }
        public function getMass(): int {
            return $this->mass;
        }

        public function calculateForce(): ?float {
            $this->inertia = $this->mass;
            return $this->inertia;
        }
    }
?>

File: poo/3. Newton's Laws Simulator  JS/inertia.java
public class Inertia extends Forces {
    private int inertia;
    private final int mass;

    public Inertia(int mass) {
        super("Inercia", "Inercia");
        this.mass = mass;
        this.inertia = mass;
    }

    public int getInertia() {
        return inertia;
    }

    public int getMass() {
        return mass;
    }

    @Override
    public Float calculateForce() {
        this.inertia = this.mass;
        return (float) inertia;
    }
}
